modules/claims_api/config/schemas/v2/526.json

```diff
@@ -147,15 +146,15 @@
                 "properties": {
                     "beginDate": {
                         "description": "Date in YYYY-MM-DD the Veteran changed address. Date must be in the future if change is temporary.",
-                        "pattern": "^$|^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$",
+                        "pattern": "^$|^(0[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])-(\\d{4})$",
                         "type": "string",
```

Comment:
In line with my above comment, changes like this will be especially important to track since it will be easy to miss these later on I suspect (at least it would be for me).

Reply:
All set.